src/evm-canister-client/src/client.rs

```diff
@@ -844,4 +844,14 @@ impl<C: CanisterClient> EvmCanisterClient<C> {
     ) -> CanisterClientResult<Result<()>> {
         self.client.update("revert_to_block", (args,)).await
     }
+
+    /// Receive a batch of  Transactions and append them to the blockchain.
+    pub async fn receive_transactions_unchecked(
```

Comment:
This is to send, not to receive

Reply:
No, the client `sends` transactions, for example, the Ethereum endpoint to send a TX is indeed called **SEND**RawTransaction. If you don't like `send`, you can find a synonym like `push` or `add` but not `receive`